.github/workflows/test-ci.yml

```diff
@@ -20,17 +20,21 @@ jobs:
       run: |
         # $CONDA is an environment variable pointing to the root of the miniconda directory
         echo $CONDA/bin >> $GITHUB_PATH
-    - name: Install conda env & dependencies
+    - name: Install conda env & dependencies on linux
+      if: ${{ matrix.os == 'ubuntu-latest' }}
       run: |
        conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch
```

Comment:
I know the conflicts and this torch libs thing, I will change that later. I was going to do it tomorrow, cuz the coverage script didn't work because of the merge from other PRs.